INSTALL.rst.txt

```diff
@@ -81,12 +78,8 @@ recent gfortran from an OS X package manager (like Homebrew).
 Please do NOT use gfortran from `hpc.sourceforge.net <http://hpc.sourceforge.net>`_,
 it is known to generate buggy scipy binaries.
 
-
-Blas/Lapack
-~~~~~~~~~~~
-
-Mac OS X includes the Accelerate framework: it should be detected without any
-intervention when building SciPy.
+You should also use a BLAS/LAPACK library from an OS X package manager.
```

Comment:
I'm not a fan of recommending that BLAS/LAPACK is obtained from a package manager. I've been burnt by package managers in the past, leaving cruft all over the place. I'd prefer that we give instructions on where to get a sufficient library from, and how to compile it. It'd drive me crazy having to install homebrew solely to compile a BLAS library.